mmdet/models/dense_heads/anchor_head.py

```diff
@@ -315,8 +316,8 @@ def get_targets(self,
             label_channels=label_channels,
             unmap_outputs=unmap_outputs)
         (all_labels, all_label_weights, all_bbox_targets, all_bbox_weights,
-         pos_inds_list, neg_inds_list) = results[:6]
-        rest_results = list(results[6:])  # user-added return values
+         pos_inds_list, neg_inds_list, sampling_results_list) = results[:7]
```

Comment:
I am afraid the returning values of get_targets_single might be incompatible with that defined in [fsaf](https://github.com/open-mmlab/mmdetection/blob/bb8458dd4c2f2cd409253df8898b6fe4a91f2953/mmdet/models/dense_heads/fsaf_head.py#L43).